.github/workflows/dotnet-build.yml

```diff
@@ -96,7 +102,57 @@ jobs:
         echo "Build AutoGen"
         dotnet build --no-restore --configuration Release -bl /p:SignAssembly=true
     - name: Unit Test
-      run: dotnet test --no-build -bl --configuration Release
+      run: dotnet test --no-build -bl --configuration Release --filter type=!integration
+  
+  integration-test:
+    strategy:
+      fail-fast: true 
+      matrix:
+        os: [ ubuntu-latest]
```

Comment:
In later PR we can add `win` in os matrix

Reply:
https://github.com/microsoft/autogen/issues/4618